src/test/resources/eval/v1.1/apps_579_boolean_flag_expr.wdl

```diff
@@ -0,0 +1,25 @@
+version 1.1
```

Comment:
@jdidion  Maybe it should be a test against 1.0 instead?

Reply:
Or should it be tested under syntax like https://github.com/dnanexus-rnd/wdlTools/pull/135/files ? They're also in dxCompiler integration tests in my other PR, here I just want to check that they compile without errors

Reply:
1.1 is fine

Reply:
@jdidion   I'm trying to add the test now. I only added the WDL files for the test so far.